lib/itoolkit.js

```diff
@@ -33,7 +33,7 @@ const xmlToJson = (xml) => {
   const pgmReg = /<pgm name='(.*?)' lib='(.*?)'.*?>/;
   
   const successReg = /<success>.*?\+\+\+ success (.*?)<\/success>/;
-  const errorReg = /<error>.*?\*\*\* error (.*?)<\/error>.*?<error>(.*?)<\/error>/;
+  const errorReg = /<error>.*?\*\*\* error (.*?)<\/error>.*?\n<error>(.*?)<\/error>/;

```

Comment:
```suggestion
  const errorReg = /<error>.*?\*\*\* error (.*?)<\/error>.*?\n?<error>(.*?)<\/error>/;
```

This will allow having a newline or not.

Reply:
Note that the `?` following the `*` doesn't mean the usual "one or more of the previous tokens", but instead modifies the `*` to make it non-greedy. This seems to be particular to Javascript. See the table here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Using_special_characters